screenpy/narration/narrator.py

```diff
@@ -205,7 +209,7 @@ def narrate(self, channel: str, **kwargs: Union[Kwargs, None]) -> ContextManager
                 (channel, channel_kws, self.exit_level)
             )
         else:
-            enclosed_func = self._entangle_func(channel, **channel_kws)  # type: ignore
+            enclosed_func = self._entangle_func(channel, None, **channel_kws)
```

Comment:
So, this one might have been an actual real issue that was being ignored by the `type: ignore`. I don't remember why i had this `type: ignore` here, but whatever the reason, it was covering up that i wasn't passing in the `adapters` list for this call.

Nice catch, `pygrep-hooks`!

Reply:
That means we didn't have a good test for kwargs?

Reply:
I think it's more a weird quirk of having a default arg followed by **kwargs, then passing in those kwargs using ** as well. I think `mypy` might misinterpret what's happening since
```python
self._entangle_func(channel, **channel_kws)
```
could actually be setting the adapters list in that explosion. 🤷 might be a bug in mypy?